exp/agents/agents.go

```diff
@@ -0,0 +1,51 @@
+// Package agents defines the types for langchaingo Agetns.
```

Comment:
please place package-level comments into a doc.go file

Reply:
also there's a typo in "Agetns"